tests/PHPUnit/Model/StartedKataTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * This file is part of the phpkata project.
+ *
+ * (c) Yannick Voyer (http://github.com/yvoyer)
+ */
+
+namespace tests\PHPUnit\Model;
+
+use Star\Kata\Model\Kata;
+use Star\Kata\Model\StartedKata;
+
+/**
+ * Class StartedKataTest
+ *
+ * @author  Yannick Voyer (http://github.com/yvoyer)
+ *
+ * @package tests\PHPUnit\Model
+ *
+ * @covers Star\Kata\Model\StartedKata
+ */
+final class StartedKataTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var StartedKata
+     */
+    private $kata;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject
+     */
+    private $wrappedKata;
+
+    public function setUp()
+    {
+        $this->wrappedKata = $this->getMockBuilder(Kata::CLASS_NAME)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->kata = new StartedKata($this->wrappedKata);
+    }
+
+    public function test_should_return_the_name()
+    {
+        $this->wrappedKata
```

Comment:
a `Kata` is something that is basic to instantiate.. have you considered using the `ObjectMother` pattern in your tests to create a real `Kata` instead of a mock? Even without using this pattern `Kata` object still doesn't seem like something that should be typically mocked.

Although this class doesn't seem very useful anyway, so that would eliminate the need for these tests altogether.